doc/upstream_release_process.md

```diff
@@ -80,6 +76,23 @@ the numbers. not done here to have you sanity check output.
 
         grep '^[ ]*LP' git-log | sed -e 's,#,,g' -e 's,.*LP: ,,' -e 's/,[ ]*/\n/'
 
+## Tag and push it to git
```

Comment:
Do we need this in doc if we are encapsulating that work in scripts/upstream-snapshot?

Reply:
These lines are dropped from `upstream-release` because they shouldn't be run until after the PR has landed in GitHub.

Reply:
Ahh right we'd lose context by the time we needed the commandline 
+1